src/timesync/timesyncd.c

```diff
@@ -149,27 +143,24 @@ int main(int argc, char *argv[]) {
                 log_warning_errno(r, "Failed to parse configuration file: %m");
 
         r = manager_parse_fallback_string(m, NTP_SERVERS);
-        if (r < 0) {
-                log_error_errno(r, "Failed to parse fallback server strings: %m");
-                goto finish;
-        }
+        if (r < 0)
+                return log_error_errno(r, "Failed to parse fallback server strings: %m");
 
         log_debug("systemd-timesyncd running as pid " PID_FMT, getpid_cached());
-        sd_notify(false,
-                  "READY=1\n"
-                  "STATUS=Daemon is running");
```

Comment:
A different approach: define
```C
#define NOTIFY_READY "READY=1\n" "STATUS=Daemon is running"
#define NOTIFY_STOPPING  "STOPPING=1\n" "STATUS=Shutting down..."
const char *notify_start(const char *start, const char *stop) {
        sd_notify(false, start);
        return stop;
}
```

And then just wrap part of code like this one in
```C
{
         _cleanup_(notify_on_cleanup) const char *status = notify_start(NOTIFY_READY, NOTIFY_STOPPING);

         r = sd_event_loop(m->event);
         if (r < 0)
                return ...;
}
```
If it is necessary to override either the starting or stopping message, they can be override in the arguments to `notify_start`, and the stop message can also be overridden by setting `status`.